packages/wallets/src/components/ErrorState/ErrorState.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import IcCashierError from '../../public/images/ic-cashier-error.svg';
+import FundsProtection from '../FundsProtection';
+import { GenericPageWithAction } from '../GenericPageWithAction';
+
+type TProps = {
+    error: unknown;
+};
+
+const ErrorState: React.FC<TProps> = ({ error }) => {
```

Comment:
Can you pls pass in code and message instead of just plain error. So that we can type it above